src/main/java/com/celements/model/classes/fields/AbstractClassField.java

```diff
@@ -19,23 +17,22 @@
  */
 public abstract class AbstractClassField<T> implements ClassField<T> {
 
-  private final DocumentReference classRef;
+  private final String className;
   private final String name;
   private final String prettyName;
   private final String validationRegExp;
   private final String validationMessage;
 
   public abstract static class Builder<B extends Builder<B, T>, T> {
 
-    private final DocumentReference classRef;
+    private final String className;
     private final String name;
     private String prettyName;
     private String validationRegExp;
     private String validationMessage;
 
-    public Builder(@NotNull DocumentReference classRef, @NotNull String name) {
-      Objects.requireNonNull(classRef);
-      this.classRef = References.cloneRef(classRef, DocumentReference.class);
+    public Builder(@NotNull String className, @NotNull String name) {
```

Comment:
please do not use String representations for document full names. We are working about 10 years to get rid of all String typed full names. Use DocumentReferences or EntityReferences instead.